astropy/visualization/wcsaxes/ticklabels.py

```diff
@@ -227,7 +231,7 @@ def draw(self, renderer, bboxes, ticklabels_bbox, tick_out_size):
                 # that has a key starting bit such as -0:30 where the -0
                 # might be dropped from all other labels.
 
-                if not self._exclude_overlapping or bb.count_overlaps(bboxes) == 0:
```

Comment:
I don't believe the bit I have removed ever returned anything but `True`, as at this point in the drawing `bboxes` was always just an empty list (so there were never any overlaps with `bb`).